src/css/components/Checkbox.scss

```diff
@@ -115,6 +120,27 @@ $Checkbox-bg-color: $cu-foreground !default;
   .Checkbox-label:before {
     border-radius: ( $Checkbox-size / 2 );
   }
+
+  .Checkbox-label:after {
+    content: '';
+    display: none;
+    height: $Checkbox-size - 4;
```

Comment:
any reason why this couldn't be $su-xsmall?

Reply:
These changes are from an earlier commit and not part of the current diff.